docs/custom.md

```
@@ -0,0 +1,156 @@
+---
+sidebar_position: 3
+---
+
+# Advanced Techniques
+
+## Custom Serializers
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)